packages/types-known/src/spec/centrifuge-chain.ts

```diff
@@ -77,10 +71,20 @@ const sharedTypes = {
 
 const versioned: OverrideVersionedType[] = [
   {
-    minmax: [240, undefined],
+    minmax: [240, 999],
     types: {
       ...sharedTypes,
+      AccountInfo: 'AccountInfoWithRefCount',
+      Address: 'LookupSource',
+      LookupSource: 'IndicesLookupSource',
+      Multiplier: 'Fixed64',
       RefCount: 'RefCountTo259'
+    },
+  },
+  {
+    minmax: [1000, undefined],
+    types: {
+      ...sharedTypes,
```

Comment:
```suggestion
      ...sharedTypes
```

Lint probably complains here (have not looked at CI...)